scripts/cloudlab/start_onenode_vhive_cluster.sh

```diff
@@ -26,6 +26,8 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT="$( cd $DIR && cd .. && cd .. && pwd)"
 SCRIPTS=$ROOT/scripts
 
+GVISOR=$1
```

Comment:
let's have SANDBOX variable that can be {stock, firecracker, gvisor} and nothing else (add an assertion). please replace the `stock-only` variable with that

Reply:
I think `stock-only` is unrelated to this script, no? Stock-only is a variable in `./scripts/cluster/create_one_node_cluster.sh`, and is used to set the socket for the cri. With gvisor, we would still use the same cri socket as used for firecracker, which is the socket where the vHive-cri is listening.

Reply:
i agree. let's keep stock separate